src/adagrams.js

```diff
@@ -1,7 +1,124 @@
+import _ from 'lodash';
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const LETTER_DISTRIB = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1
+    };
+
+    let letterPool = [];
+
+    for (const letter in LETTER_DISTRIB) {
+      for (let i = 0; i < LETTER_DISTRIB[letter]; i++) {
+        letterPool.push(letter);
+      }
+    }
+    
+    // Fisher-Yates shuffle algorithm
```

Comment:
I know you didn't end up using this but great that you gave credit here! In addition, assuming you found the code from the web somehere, it can be good to link to the webpage where you found the code.